### PR TITLE
MODQM-449 Update source-manager-job-executions and field-protection-settings versions

### PR DESCRIPTION
**Pull request by dmytrokrutii**
### Purpose
According to the epic [FOLIO-4044](https://folio-org.atlassian.net/browse/FOLIO-4044), some permissions have been changed in the source-manager-job-executions `3.4` and field-protection-settings `1.2`

### Approach
Update required modules versions and rename permissions

Permission | API
-- | --
"converter-storage.field-protection-settings.collection.get" | /field-protection-settings/marc"
"converter-storage.field-protection-settings.item.get" | /field-protection-settings/marc/{id}
"change-manager.jobExecutions.jobProfile.item.put" | /change-manager/jobExecutions/{id}/jobProfile

### Changes Checklist
- [ ] **API Changes**: Document any API paths, methods, request or response bodies changed, added, or removed.
- [ ] **Database Schema Changes**: Indicate any database schema changes and their impact. Confirm that migration scripts were created.
- [ ] **Interface Version Changes**: Indicate any changes to interface versions.
- [ ] **Interface Dependencies**: Document added or removed dependencies.
- [x] **Permissions**: Document any changes to permissions.
- [ ] **Logging**: Confirm that logging is appropriately handled.
- [ ] **Unit Testing**: Confirm that changed classes were covered by unit tests.
- [ ] **Integration Testing**: Confirm that changed logic was covered by integration tests.
- [ ] **Manual Testing**: Confirm that changes were tested on local or dev environment.
- [x] **NEWS**: Confirm that the NEWS file is updated with relevant information about the changes made in this pull request.

### Related Issues
[MODQM-449](https://folio-org.atlassian.net/browse/MODQM-449)
[MODDICONV-393](https://folio-org.atlassian.net/browse/MODDICONV-393)
[MODSOURMAN-1222](https://folio-org.atlassian.net/browse/MODSOURMAN-1222)

### Learning and Resources (if applicable)
[FOLIO-4044](https://folio-org.atlassian.net/browse/FOLIO-4044)

### CODE REVIEW

**Review by psmagin — COMMENTED** [review]
Please provide more details about a change in both PR description and related Jira issue.
Please provide more meaningful and specific commit message that will represent the change (at least in commit body)
Please update NEWS

**Comment by dmytrokrutii** [comment]
> Please provide more details about a change in both PR description and related Jira issue. Please provide more meaningful and specific commit message that will represent the change (at least in commit body) Please update NEWS

I'm sorry. I forgot to mark the PR as a draft.
I'll mark the PR as ready for review when I make the appropriate changes

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -51,7 +51,8 @@
           "modulePermissions": [
             "change-manager.parsedrecords.get",
             "users.item.get",
-            "converter-storage.field-protection-settings.get",
+            "converter-storage.field-protection-settings.item.get",
+            "converter-storage.field-protection-settings.collection.get",
```
Comment (psmagin) [inline descriptors/ModuleDescriptor-template.json:55]: We use only `field-protection-settings/marc` endpoint, so I believe that we should have only "converter-storage.field-protection-settings.collection.get" permission. Am I right?

**Review by KaterynaSenchenko — APPROVED** [review]

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (dmytrokrutii) [reply descriptors/ModuleDescriptor-template.json]: Indeed, in the `FieldProtectionSettingsClient` interface, there is only `field-protection-settings/marc`. Thanks for the comment. I fixed it.

**Review by psmagin — APPROVED** [review]

**Review by viacheslavkol — APPROVED** [review]

**Review by SvitlanaKovalova1 — APPROVED** [review]

**Pull request merged**